test/parallel/test-event-emitter-errors.js

```diff
@@ -7,4 +7,8 @@ var EE = new EventEmitter();
 
 assert.throws(function() {
   EE.emit('error', 'Accepts a string');
-}, /Accepts a string/);
+}, /Unhandled "error" event/);
```

Comment:
Nit: how about using `^Error: Unhandled "error" event$`? Pointing this out only because other tests do this.